src/types.ts

```diff
@@ -65,6 +65,14 @@ export interface SessionOptions {
    * @docs https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#restrict_access_to_cookies
    */
   cookieSecure: boolean
+  /**
+   * Wether to set the `HttpOnly` attribute for the session cookie
```

Comment:
I didn't catch this in the previous review, but: Could you also add the "why?" to this doc-string?

Eg:
```suggestion
   * Wether to set the `HttpOnly` attribute for the session cookie. When `HttpOnly` is set the session cookie will not be accessible from JavaScript, this can mitigate XSS attacks
```